src/main/java/com/github/redouane59/twitter/helpers/AbstractRequestHelper.java

```diff
@@ -29,19 +26,21 @@
 
   private final TwitterCredentials twitterCredentials;
   private final OAuth10aService service;
+  protected IAPIEventListener listener = null;
+  protected final TweetStreamConsumer tweetStreamConsumer;
   
-  public AbstractRequestHelper(TwitterCredentials twitterCredentials) {
+  protected AbstractRequestHelper(TwitterCredentials twitterCredentials) {
 	  this(twitterCredentials, new ServiceBuilder(twitterCredentials.getApiKey())
 			  .apiSecret(twitterCredentials.getApiSecretKey())
 			  .build(TwitterApi.instance()));
   }
   
-  public AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aService service) {
+  protected AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aService service) {
 	  this.twitterCredentials = twitterCredentials;
 	  this.service = service;
+    this.tweetStreamConsumer = new TweetStreamConsumer();
```

Comment:
why don't you instantiate it directly above ? 
Is it normal that the other constructor `AbstractRequestHelper(TwitterCredentials twitterCredentials)` doesn't instantiate it ?

Reply:
AbstractRequestHelper(TwitterCredentials twitterCredentials) calls AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aService service). So I thought it was best to put it here ;)

Reply:
Oh yes, forgot it you're right I didn't see the super() call in the other constructor lol